src/main/java/com/facebook/genAlgo/evaluator/EvaluatorFactory.java

```diff
@@ -9,9 +9,15 @@
     }
 
     Evaluator getEvaluator(char target) {
-        return null;
+        return getEvaluator(target, EvaluatorEnum.DEFAULT);
     }
+
     Evaluator getEvaluator(char target, EvaluatorEnum option) {
-        return null;
+        Evaluator evaluator = null;
+        switch (option) {
+            case DEFAULT, MAX_DELTA -> evaluator = new MaxDeltaEvaluatorImpl(target);
```

Comment:
I am wondering if it is better to reduce this to one if because only one option gets LogEvaluator and everything else is default evaluator.

Reply:
right, I will change it